source/Calamari.Terraform.Tests/CommandsFixture.cs

```diff
@@ -26,13 +26,12 @@
 
 namespace Calamari.Terraform.Tests
 {
-    [TestFixture("0.11.15")]
-    [TestFixture("0.13.0")]
-    [TestFixture("1.0.0")]
+    [TestFixture("0.13.7")]
+    [TestFixture("1.8.5")]
```

Comment:
After chatting with @zentron, our oldest supported version is `0.13` series, as it appears to not be EOL (yet)